Misc/NEWS.d/next/Library/2019-03-27-15-09-00.bpo-35900.fh56UU.rst

```diff
@@ -0,0 +1,2 @@
+Allow the user to specify custom functions to set the state of an object in
+save_reduce via the new state_setter keyword argument.
```

Comment:
"save_reduce" and "state_setter" are implementation details, so perhaps phrase this differently?
For example:
```rst
Allow reduction methods to return a 6-item tuple where the 6th item specifies a custom state-setting method that's called instead of the regular ``__setstate__`` method.
```